app/src/main/res/drawable/ic_favorite_filled_24dp.xml

```diff
@@ -5,5 +5,5 @@
         android:viewportHeight="24.0">
     <path
         android:pathData="M12,21.35l-1.45,-1.32C5.4,15.36 2,12.28 2,8.5 2,5.42 4.42,3 7.5,3c1.74,0 3.41,0.81 4.5,2.09C13.09,3.81 14.76,3 16.5,3 19.58,3 22,5.42 22,8.5c0,3.78 -3.4,6.86 -8.55,11.54L12,21.35z"
-        android:fillColor="@color/song_row_favorite_color"/>
+        android:fillColor="#FF6E40"/>
```

Comment:
It seems that color/song_row_favorite_color is still "#FF6E40" in this CL. Am I wrong?